src/lib/y2storage/planned/device.rb

```diff
@@ -60,8 +61,39 @@ def self.to_string_attrs
         [:reuse]
       end
 
+      # Returns the (possibly processed) device to be used for the planned
+      # device.
+      #
+      # FIXME: temporary API. It should be improved.
+      def final_device!(plain_device)
+        plain_device.encrypted? ? plain_device.encryption : plain_device
```

Comment:
Yes, that's why there is a `FIXME: temporary API. It should be improved.` comment there. We plan to do it while adding support for LVM to the AutoYaST proposal.